packages/node_modules/@webex/internal-plugin-mercury/src/mercury.js

```diff
@@ -167,6 +167,12 @@ const Mercury = WebexPlugin.extend({
 
     Promise.all([this._prepareUrl(socketUrl), this.webex.credentials.getUserToken()])
       .then(([webSocketUrl, token]) => {
+        if (!this.backoffCall) {
+          this.logger.info('mercury: prevent socket open when backOffCall no longer defined');
+
+          return Promise.reject(new Error('mercury: prevent socket open when backOffCall no longer defined due to abort?'));
```

Comment:
Is there a reason why the error message is slightly different than the one just above it? Could we just reuse the string?

Reply:
No good reason - I've updated to make consistent